src/librustc_middle/query/mod.rs

```diff
@@ -89,6 +89,20 @@ rustc_queries! {
             desc { |tcx| "HIR owner items in `{}`", tcx.def_path_str(key.to_def_id()) }
         }
 
+        /// Computes the `DefId` of the corresponding const parameter of a const argument.
+        /// Returns `None` if `def_id` is not a const argument.
```

Comment:
Is this expected to be called for non-const-argument `def_id`s?

Reply:
Yes, made this more explicit in the query description.